recipes/mlrose/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "mlrose" %}
+{% set version = "1.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: cec83253bf6da67a7fb32b2c9ae13e9dbc6cfbcaae2aa3107993e69e9788f15e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3
+    - pip
+  run:
+    - numpy
+    - scipy
+    - scikit-learn <0.21.0
+    - python >=3
+
+test:
+  imports:
+    - mlrose
+  requires:
+    - pip
+  commands:
+    - pip check
```

Comment:
The python package still requires `sklearn` instead of `scikit-learn`. This is why the test is failing. Remove this check (and the requires pip).
The import will be enough.